### PR TITLE
One of with strings bug

### PR DESCRIPTION
**Pull request by zmedelis**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,6 @@ run-eval:
 	docker-compose -p dev -f docker-compose.yml -f docker-compose.eval.yml up --remove-orphans --abort-on-container-exit --exit-code-from eval
 
 run-front-end-dev:
-	ACC_TEXT_API_URL=http://0.0.0.0:3001 \
-	ACC_TEXT_GRAPHQL_URL=http://0.0.0.0:3001/_graphql \
-	MOCK_SHOP_API_URL=http://0:0:0:0:8090 \
-	cd front-end && make run
+	docker-compose -f docker-compose.front-end.yml up -d
+	cd core/gf && docker build -t gf .
+	docker run -p 8001:8000 gf
```
Comment (zaibacu) [inline Makefile:55]: Need to build docker-compose file, this `docker run` will cause many problems

File: core/test/acc_text/nlg/core_test.clj [hunk 2]
```diff
@@ -80,3 +80,9 @@
   (let [semantic-graph (test-utils/load-test-semantic-graph "parallel-edge-test")
         context (test-utils/load-test-context "parallel-edge-test")]
     (is (= ["There is an item."] (map :text (core/generate-text semantic-graph context "Eng"))))))
+
+;; (deftest ^:integration one-of-generation
+;;   (let [semantic-graph (test-utils/load-test-semantic-graph "one-of-with-str")
+;;         context        (test-utils/load-test-context "one-of-with-str")]
+;;     (is (= #{}
+;;            (core/generate-text semantic-graph context "Eng")))))
```
Comment (zaibacu) [inline core/test/acc_text/nlg/core_test.clj:88]: uncomment/delete?

Comment (zmedelis) [reply core/test/acc_text/nlg/core_test.clj]: Fixed

File: Makefile (hunk 1 above)
Comment (zmedelis) [reply Makefile:55]: Let's have it as is and  review. Don't think we will be using this command anyways.

**Comment by rokasramas** [comment]
Some nodes get unattached, in the example below - 'the' is missing. 

Screenshot from 2020-04-23 17-06-27
Screenshot from 2020-04-23 17-06-40

**Comment by rokasramas** [comment]
Need to make sure that synonym node and edge categories are ok
Screenshot from 2020-04-25 20-36-37

**Pull request merged**